packages/open-payments/src/client/index.ts

```diff
@@ -19,6 +20,8 @@ import { AxiosInstance } from 'axios'
 export interface CreateOpenPaymentClientArgs {
   requestTimeoutMs?: number
   logger?: Logger
+  privateKey: KeyLike
+  keyId: string
```

Comment:
- https://github.com/interledger/rafiki/issues/731

Thinking about :point_up_2:, if there are clients that only want to use `open-payments` to query payment pointers and/or [their keys](https://docs.openpayments.guide/reference/get-payment-pointer-keys) and/or connections, it seems like it should be possible to create a client without requiring `privateKey`/`keyId`.

Reply:
Should `open-payments` `createClient` be split into:
- `createAuthenticatedClient`
- `createUnauthenticatedClient`
?
The latter would return the subset of requests that are possible without httpsig.